hw_19.09.23/task1.py

```diff
@@ -0,0 +1,39 @@
+class Person:
+    def __init__(self, first_name, last_name):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.full_name = first_name + " " + last_name
+
+
+class Student(Person):
+    def __init__(self, attendance, assessments, first_name, last_name):
```

Comment:
If at some point you decide to modify/extend parameters in `Person`'s `__init__` - you'll need to modify them in three places at once (in `Person`, here and below in `Teacher`). It would be better to use `*args, **kwargs``